jetty-server/src/main/java/org/eclipse/jetty/server/session/AbstractSessionCache.java

```diff
@@ -111,12 +111,13 @@
     public abstract Session newSession(HttpServletRequest request, SessionData data);
 
     /**
-     * Get the session matching the key
+     * Get the session matching the key from the cache. Does not load
+     * the session.
      *
      * @param id session id
      * @return the Session object matching the id
      */
-    public abstract Session doGet(String id);
+    protected abstract Session doGet(String id);
```

Comment:
changing visibility in minor release? (maybe in 10?)

Reply:
Yes, bit naughty I know. These methods should have been protected from the start. I figure that as they are only called by the session cache itself, then there should be no incompatibilities with this change.